pkg/entrypoint/etcdbackup/cmd.go

```diff
@@ -28,16 +27,16 @@ func NewCommand() *cobra.Command {
 			return start(cfg)
 		},
 	}
-	cc.Flags().String("blobname", "", "Name of the blob (without the container)")
+	cc.Flags().String("blobName", "", "Name of the blob (without the container)")
 	cc.Flags().String("destination", "", "Where to place the blob on the filesystem")
 	cc.Flags().Int("maxBackups", 6, "Maximum number of backups to keep")
 	cc.Flags().StringP("action", "a", "save", "Action to be executed [save, download]")
 
 	return cc
 }
 
-func configFromCmd(cmd *cobra.Command) (*Config, error) {
-	c := &Config{}
+func configFromCmd(cmd *cobra.Command) (*commandConfig, error) {
```

Comment:
`Config` is the same struct all around the `cmd` files for all `entrypoint/{entrypoint}/Config` files. So Config kidna fits the name here.
https://github.com/openshift/openshift-azure/blob/master/pkg/entrypoint/metricsbridge/cmd.go#L9
https://github.com/openshift/openshift-azure/blob/master/pkg/entrypoint/startup/cmd.go#L9
etc. Which makes boiler-plate same. Either rename it everywhere to `cmdConfig` or rollback the change.

Reply:
you are right, pushed 6ae2ded0fd76b88e1d9b11368c5ad122dbe78617